t/32-openqa_client.t

```diff
@@ -191,11 +193,11 @@ subtest 'upload failures' => sub {
             is(pop()->res->json->{status}, 'foobar', 'Error message status is correct');
         });
 
-    eval {
+    lives_ok {
         $t->ua->upload->asset(99963 =>
               {chunk_size => $chunk_size, file => $filename, name => 'hdd_image5.xml', asset => 'other', retries => 7});
-    };
-    ok !$@, 'No function errors on upload failures' or die diag $@;
+    }
+    'No function errors on upload failures';
```

Comment:
The original code has a `die` here. I guess it's not that important and was there simply to prevent executing the rest of the tests. Same below. Maybe just use `BAIL_OUT`?

Reply:
I hope I did it right

Reply:
If you want to find out whether you did it right you can (temporarily) insert a `die` in the `lives_ok` block and see whether the test bails with the die message you put in.

Reply:
Well, there are several other occurrences. In some you removed the `die`, in some you kept it, and in some you replaced it by `BAIL_OUT` now.
I think it would be good to be consistent, but others have approved it so I guess it's not important enough.

Reply:
@b10n1k It shouldn't be too much work so can you make it consistent like @perlpunk suggested?

Reply:
Apparently @b10n1k did not see these comments after merging.
I now added explicit comments on every occurrence.

Reply:
https://github.com/os-autoinst/openQA/pull/6274